### PR TITLE
WebConsole safe temp fix: enable auth for egress streaming http request

### PR DESCRIPTION
**Pull request by Karakatiza666**
This should be robust enough until the proper fix that would directly use method generated from OpenAPI spec and build on top of it.

### CODE REVIEW

File: web-console/src/lib/compositions/streaming/inspection/useTableUpdater.ts [hunk 1]
```diff
@@ -79,21 +82,24 @@ export function useTableUpdater() {
...
-      requestedNeighborhood: NeighborhoodQuery,
+      egressParams: Arguments<typeof HttpInputOutputService.httpOutput>,
       setRows: Dispatch<SetStateAction<Row[]>>,
       setLoading: Dispatch<SetStateAction<boolean>>,
       relation: Relation,
       controller: AbortController
     ) => {
       // We try and fetch one more row than requested,
       // this helps to detect if this is the last page.
+      // TODO:
```
Comment (mihaibudiu) [inline web-console/src/lib/compositions/streaming/inspection/useTableUpdater.ts:93]: isn't it enough to have this comment once in the other file?

File: web-console/src/lib/components/streaming/inspection/InspectionTable.tsx [hunk 2]
```diff
@@ -99,16 +99,22 @@ const useInspectionTable = ({ pipeline, name }: InspectionTableProps) => {
...
-        '/v0' +
-        '/pipelines/' +
-        pipeline.descriptor.pipeline_id +
-        '/egress/' +
-        name +
-        '?format=csv&query=neighborhood&mode=watch'
-    )
-    updateTable(url, neighborhood, setRows, setLoading, relation, controller).then(
+    updateTable(
+      [
```
Comment (mihaibudiu) [inline web-console/src/lib/components/streaming/inspection/InspectionTable.tsx:103]: I am a little puzzled by this signature.
The first argument is an array of ANY?
Can this be stronger-typed?

File: web-console/src/lib/compositions/streaming/inspection/useTableUpdater.ts (hunk 1 above)
Comment (Karakatiza666) [reply web-console/src/lib/compositions/streaming/inspection/useTableUpdater.ts:93]: I wanted to reeeally make sure this is not missed

Comment (mihaibudiu) [reply web-console/src/lib/compositions/streaming/inspection/useTableUpdater.ts:93]: But when this is fixed, does this place need to be corrected, or just the implementation?
If the latter, only one comment is enough.

File: web-console/src/lib/components/streaming/inspection/InspectionTable.tsx (hunk 2 above)
Comment (Karakatiza666) [reply web-console/src/lib/components/streaming/inspection/InspectionTable.tsx:103]: My type-checker works as expected:
image
enabled by 
image

This change is final, it is designed to follow dependency injection principle (I think it's a good fit here)
So the only thing to be updated later is, in this case, implementation of `updateTable` function

Comment (Karakatiza666) [reply web-console/src/lib/components/streaming/inspection/InspectionTable.tsx:103]: image

Comment (mihaibudiu) [reply web-console/src/lib/components/streaming/inspection/InspectionTable.tsx:103]: ah, it's a tuple, not an array

**Review by mihaibudiu — APPROVED** [review]

**Pull request merged**